support-services/src/main/scala/com/gu/support/catalog/CatalogService.scala

```diff
@@ -77,10 +86,11 @@ class CatalogService(val environment: TouchPointEnvironment, jsonProvider: Catal
     currency: Currency,
     billingPeriod: BillingPeriod,
     fulfilmentOptions: FulfilmentOptions,
-    productOptions: ProductOptions
+    productOptions: ProductOptions,
+    readerType: ReaderType = Direct
   ): Option[Price] = {
     for {
-      productRatePlan <- product.getProductRatePlan(environment, billingPeriod, fulfilmentOptions, productOptions)
+      productRatePlan <- product.getProductRatePlan(environment, billingPeriod, fulfilmentOptions, productOptions, readerType)
```

Comment:
was this a bug before? the missing readerType.

Reply:
I've removed this method because it was only used by test code. It's now in CatalogServiceSpec